rdm/0004-browser-support.md

```diff
@@ -0,0 +1,77 @@
+- Start Date: 2019-10-02
+- RFC PR: [5](https://github.com/inveniosoftware/rfcs/pull/5)
+- Authors: Guillaume Viger
+
+# Support Evergreen Browsers
+
+## Summary
+
+InvenioRDM commits to supporting secure, evergreen browsers that are themselves
+supported by their vendors.
+
+
+## Motivation
+
+As an InvenioRDM core or extension/plugin developer, I want to take advantage
+of modern Web APIs such as flexbox, websockets, File API, modern Javascript (ES2015+)...
+so that I can provide more powerful features at reduced development and maintenance
+(polyfills...) cost.
+
+As an InvenioRDM hoster, running an application that requires evergreen secure
+browsers benefits me since it ensures my users are running secure browsers
+which reduces my surface of attack and reduces my support costs because evergreen
+browsers continually fix their bugs and security holes.
+
+
+## Detailed design
+
+Evergreen Microsoft Edge, Google Chrome, Mozilla Firefox, Apple Safari are
```

Comment:
I would add a caveat that this policy should be reviewed after review needs for the React applications that are going to be built.

Reply:
Yes, it would be good to see how this works in practice when developing with a framework with its own support.